README.md

```diff
@@ -56,7 +56,19 @@ puts lc_vers.version_string # => "10.10.0"
 
 * Unit and performance testing.
 
-Attribution:
+### Contributing, setting up `overcommit` and the linters
+
+In order to keep the repo, docs and data tidy, we use a tool called [`overcommit`](https://github.com/sds/overcommit)
+to connect up the git hooks to a set of quality checks.  The fastest way to get setup is to run the following to make
+sure you have all the tools:
+
+```shell
+gem install overcommit bundler
```

Comment:
Might be misunderstanding, but doesn't `bundle install` below ensure that `overcommit` is installed? So this could more tersely be:

```
gem install bundler
bundle install
bundle exec overcommit --install
```

Let me know if I'm wrong about that.

Reply:
Dang - technically you right - it just matters if you want "PostCheckout" to work after a clone or if you can do the `bundle install` first

Reply:
Gotcha, makes sense. We can leave it like this then, thanks for the explanation.